packages/elements/src/photon-med-search/photon-med-search-component.tsx

```diff
@@ -77,7 +77,7 @@ customElement(
 
     onMount(async () => {
       const { data } = await client!.apollo.query({ query: GET_CATALOGS });
```

Comment:
@marisbest2 @jknapp25 

OH it's because `GET_CATALOGS` above looks like:
```
const GET_CATALOGS = gql`
  query GetProducts($medId: String!) {
    medicationProducts(id: $medId) {
      id
      name
      controlled
    }
  }
`;
const GET_PRODUCTS = gql`
  query GetProducts($medId: String!) {
    medicationProducts(id: $medId) {
      id
      name
      controlled
    }
  }
`;
```
which is an exact copy of the one below it.... there isn't a catalogs in the query   :|